### PR TITLE
2015 06 cc section jump to core

### PR DESCRIPTION
**Pull request by local-girl**
Sorry the commits are a bit mixed up, best to look at the overview

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/*doc
+---
+title: Section Jump
+name: section-jump
+category: Widgets
+---
+
+A module with a header, a left navigation of jump links and a center section
+of chapters. Often used fo navigating proposal details
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss]: typo: for

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+```html_example
+<div class="section-jump" id="section-jump-top">
+    <div class="section-jump-cover">
+        <h1 class="section-jump-cover-header">Section Jump header</h1>
+    </div>
+
+    <div class="section-jump-body">
+        <nav class="section-jump-navigation m-unnumbered" data-adh-sticky="">
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss]: I think I would prefer not having a specific `section-jump-navigation`. Do you think it is possible to reuse the plain `jump-navigation` here?

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            margin: 0 0 1em;
+        }
+    }
+
+    @media print {
+        display: none;
+    }
+}
+
+.section-jump-chapters {
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss]: I don't like this name. IMHO it is either a section or a chapter. Using both terms is confusing.

**Comment by local-girl** [comment]
although jump navigation is extended in the stylesheet

**Comment by local-girl** [comment]
LGTM

**Pull request merged**